book/visual-effects.md

```diff
@@ -1079,31 +1079,32 @@ which describes the opacity as a floating-point number from 0 to 1.
 Note that `saveLayer` and `restore` are like a pair of parentheses
 enclosing the child drawing operations. This means our display list is
 no longer just a linear sequence of drawing operations, but a tree. So
-in our display list, let's represent `saveLayer` with a `SaveLayer`
+in our display list, let's handle `opacity` with an `Alpha`
 command that takes a sequence of other drawing commands as an
 argument:
 
-``` {.python expected=False}
-class SaveLayer:
-    def __init__(self, sk_paint, children):
-        self.sk_paint = sk_paint
+``` {.python}
+class Alpha:
```

Comment:
Sorry for the late idea but should we call it Opacity instead of Alpha? Alpha is not the same as opacity..

Reply:
Sure, but then OpacityAndBlend sounds bad, can I just call the final thing Blend?

Reply:
Sounds good!